pyopencl/cache.py

```diff
@@ -88,17 +88,18 @@ def __init__(self, cleanup_m, cache_dir):
                 except OSError:
                     pass
 
+                wait_time_seconds = 0.05
                 from time import sleep
-                sleep(0.05)
+                sleep(wait_time_seconds)
 
                 attempts += 1
 
-                if attempts > 10:
+                if attempts % (10/wait_time_seconds) == 0:
```

Comment:
That won't work reliably. `/` unconditionally has a floating point result, and comparing floating point numbers to zero is unreliable.

Reply:
> That won't work reliably. `/` unconditionally has a floating point result, and comparing floating point numbers to zero is unreliable.

4d13b82a0c387901783ccf8c2b61eacaaddec7be should fix this